Makefile

```diff
@@ -4,6 +4,7 @@ bindir ?= $(prefix)/bin
 .PHONY: develop
 develop:
 	swift package generate-xcodeproj --xcconfig-overrides settings.xcconfig
+	(killall -9 Xcode && sleep 0.25) || true # Kills Xcode and waits for the process to terminate if Xcode is open.
```

Comment:
Should we move this to _before_ we generate the project and remove the sleep?

Reply:
I'd be down to kill Xcode first, but it also feels reasonable to not kill Xcode if we fail to generate the project?

Reply:
Good point. I'm a bit hesitant about this change as is because this would kill any Xcode project someone might have open. Do you think there's:

1) An option to only kill the process that has the SwiftInspector Xcode project?
2) Use environment variables to have a way to opt-out of this?

Reply:
Those Xcode projects would re-open, so at least there's that. But there's no way to only kill the part of the Xcode process that has opened/cached the xcodeproj.

Environment variables (or a new command?) to opt out seems like a possibility.

From a usage POV: every time I run this command and it _doesn't_ do this I end up a little sad 😅

Reply:
Let's merge this as is in the meantime and we can figure out a better usage if it becomes a problem.

Reply:
Excited to get your feedback after using it a bit 😄